astropy/utils/data.py

```diff
@@ -1765,89 +1610,135 @@ def check_download_cache(check_hashes=False, pkgname='astropy'):
 
     Parameters
     ----------
-    check_hashes : bool, optional
-        Whether to compute the hashes of the contents of all files in the
-        cache and compare them to the names. This can take some time if
-        the cache contains large files.
+    check_hashes : boolean, optional
+        Deprecated; does nothing.
+
     pkgname : `str`, optional
         The package name to use to locate the download cache. i.e. for
         ``pkgname='astropy'`` the default cache location is
         ``~/.astropy/cache``.
 
     Returns
     -------
-    strays : set of strings
-        This is the set of files in the cache directory that do not correspond
-        to known URLs. This may include some files associated with the cache
-        index.
+    strays : set
+        Deprecated. The empty set.
 
     Raises
     ------
     CacheDamaged
         To indicate a problem with the cache contents; the exception contains
-        a .bad_url or a .bad_file attribute to allow the user to use
-        `clear_download_cache` to remove the offending item.
+        a ``.bad_files`` attribute containing a set of filenames to allow the
```

Comment:
Another behavior change but I guess it is acceptable given this is an attribute inside an error message that rarely happens. Does this need a change log or is it overkill?

Reply:
It seems like overkill to me.